core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.scss

```diff
@@ -1,5 +1,15 @@
 @use "variables" as *;
 
+:host {
+    display: flex;
+    height: 100%;
+    overflow: auto;
+
+    dot-portlet-base {
+        width: 100%;
```

Comment:
this should go in  `dot-portlet-base.component.scss` ? adding it here to just affect this portlet.